src/Stryker.Core/Stryker.Core/Reporters/ConsoleReportReporter.cs

```diff
@@ -95,9 +100,19 @@ private void DisplayComponent(IReadOnlyInputComponent inputComponent)
                 {
                     _chalk.Yellow(scoreText);
                 } 
-                else
+                else if (score <= thresholdLow && score >= thresholdBreak)
                 {
                     _chalk.Red(scoreText);
+                } 
+                else 
+                {
+                    // Threshold break value has been reached, set a non-zero exit code
+                    _logger.LogError("Final mutation score " 
+                        + score + "under breaking threshold " 
+                        + this.options.ThresholdOptions.ThresholdBreak 
+                        + " , setting exit code to 1 (failure)");
+                    _logger.LogInformation("Improve mutation score or set '--thresholds-break' lower to prevent this error in the future");
+                    Environment.ExitCode = 1;
```

Comment:
U would think the cli determines the exit code since the cli is the process running the test. Core should only calculate the value and determine that the threshold has not been met. This value can then be used by various reporters or runners to do what they want to do with it. An exitcode is very clearly something that belongs to the cli process, but for example the jsonreporter would report the failure in a different way, also using the values calculated by core.

Reply:
That would mean stryker.core should return some type of `StrykerRunResult`? That would make integrationtesting more easy too. So this would be great. @Stukongeluk can you do this?

Reply:
For now I think that is the way to go yes. In the future we might want to look into some lifecycle management so we can create lifecycle hooks so reporters and the like can hook into the testrun lifecycle but for now I think that is way too big.

Reply:
Thanks guys! I'm on it ;)